root/static/scripts/common/utility/sortByEntityName.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * @flow
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import {compare} from '../i18n';
+
+function compareEntities(a: CoreEntityT, b: CoreEntityT): number {
+  const nameComparison = compare(a.name, b.name);
+  if (nameComparison === 0) {
+    return a.id < b.id
+      ? -1
+      : (a.id > b.id ? 1 : 0);
+  }
+  return nameComparison;
```

Comment:
If `a.id` and `b.id` are always defined, the standard way to compare numbers is just to subtract them
```suggestion
  return compare(a.name, b.name) || (a.id - b.id);
```

Reply:
Unrelated, but we may also want to tweak this to use the sort name for artists. In my relationship editor rewrite branch I have a file at root/utility/getSortName.js which does this: https://gist.github.com/mwiencek/e7a2946edc11de6681760197dcd0eb39

So if you'd like you can copy that file to this branch and do `compare(getSortName(a), getSortName(b))` here instead. Or we can fix that later if you want to leave it for now

Reply:
I did that, but changed it to actually check entities.json for sort_name: true, in case we would ever add sort names to something else than artist :)